packages/vite-plugin-svelte/src/preprocess.ts

```diff
@@ -7,6 +6,8 @@ import type { Preprocessor, PreprocessorGroup } from 'svelte/types/compiler/prep
 const supportedStyleLangs = ['css', 'less', 'sass', 'scss', 'styl', 'stylus', 'postcss', 'sss'];
 const supportedScriptLangs = ['ts'];
 
+import { mapSourcesToRelative } from './utils/sourcemaps';
```

Comment:
Would be nice to move this up together with the other imports.

Reply:
done